src/NuGet.Core/NuGet.Packaging/PublicAPI/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,3 @@
+NuGet.Packaging.PackageBuilder.Aliases.get -> System.Collections.Generic.IDictionary<string, string>
+NuGet.Packaging.PackageBuilder.Aliases.set -> void
+static NuGet.Packaging.Rules.AnalysisResources.InvalidUndottedFrameworkWarning.get -> string
```

Comment:
out of scope for this PR, and I know you didn't do it, but why is AnalysisResources is public? 😞